Algorithm.Framework/Portfolio/EqualWeightingPortfolioConstructionModel.py

```diff
@@ -101,12 +116,17 @@ def CreateTargets(self, algorithm, insights):
         for symbol, g in groupby(activeInsights, lambda x: x.Symbol):
             lastActiveInsights.append(sorted(g, key = lambda x: x.GeneratedTimeUtc)[-1])
 
-        # Determine target percent for the given insights
-        percents = self.DetermineTargetPercent(lastActiveInsights)
+        # Determine target percent for the given insights that respect the portfolio bias
+        percents = self.DetermineTargetPercent(list(filter(self.RespectPortfolioBias, lastActiveInsights)))
```

Comment:
nit - just checking, do we need to call `list()` here?

Reply:
Yes. Python `[filter](https://docs.python.org/3/library/functions.html#filter)` returns an "iterator" that pythonnet doesn't convert into a C# List.